### PR TITLE
Improve island overview

### LINKED ISSUE #17: Improve island overview
The island overview in the UI is very basic at the moment.

I think it should:

- Show the island key, local name, local writer alias and key
- Allow to open the island
- Allow to share / unshare the island
- Sources of the islands with key writable yes/no, alias, length & size (bytes and blocks)
- Network status, number of peers

Some of this should be in the main list, some of this possibly on a details page or a collapsible panel.

The UI calls `client.status()`, which leads to `store.status()` in the backend. This function returns info on all islands. Currently it calls `this.network.status()` to collect basic networking status, and returns the list of islands from the store config. Likely we also want an `island.status` to collect info of each island.

### PR DESCRIPTION
**Pull request by hng**
The goal of this PR is to resolve #17 

It adds:
* Local key and local drive key to the overview
* Number of peers to the overview
* Share/unshare functionality
  * This introduces a new route PATCH /:islandKey which I have named change() in the client
* Adds a alias to create new island form
* Removes message above forms as we already have a notification is island creation worked or not.

### CODE REVIEW

**Comment by hng** [comment]
@Frando What is the "local writer alias and key" mentioned in #17 ?

**Comment by hng** [comment]
Still missing from #17: Sources of the islands with key writable yes/no, alias, length & size (bytes and blocks)

File: sonar-client/lib/client.js [hunk 1]
```diff
@@ -203,6 +203,15 @@ module.exports = class SonarClient {
     })
   }
 
+  async change (config, key) {
```
Comment (Frando) [inline sonar-client/lib/client.js]: I think this needs a context, `client.change()` is too ambigous. Maybe `client.updateIsland()` or `changeIsland`? We didn't really use `change` in methods much. Don't really mind too much I think. I'd likely go with `updateIsland`.

**Comment by Frando** [comment]
The local writer alias and key is, I think, not currently exposed (it's the key of the local writer, and its alias. Currently every island has a local writer, I want to change that though so that it becomes a manual operation to create a local writer for cloned islands. For locally created islands, the local writer is the only source at the beginning. Also - not sure about the terminology "local writer", I'm very open to otther suggestions). 

Maybe the best way is to expose a list of all sources, we'd want this anyway. For each source, we'd then have some info (key, alias) and some status (length, byte length, last message timestamp maybe, etc). This would be exposed by `sonar-dat/lib/island` in a new `status` method for each island I'd say. Maybe some part of it could/should be coming directly from a not-yet-existing kappa-record-db `status()` method. And we'd have to handle hyperdrives slightly different. So I think a status method on the island that collects all the info is fine for now.

In the UI, we'd have to think what we actually want to display from all the sources, not yet sure about that.

File: sonar-client/lib/client.js (hunk 1 above)
Comment (hng) [reply sonar-client/lib/client.js]: renamed function to updateIsland

**Pull request merged**